app/backend/aquifers/constants.py

```diff
@@ -0,0 +1,15 @@
+"""
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+
+AQUIFER_ID_FOR_UNCORRELATED_WELLS = 1143
```

Comment:
is this an aquifer used by the business area for this purpose?  It seems to me that there are better ways to approach this, but I can understand if this is the way it's currently handled and it's out of our control. If so maybe a comment or link to the JIRA ticket would help future devs understand what this means.

Reply:
There is a task for this in JIRA: https://apps.nrs.gov.bc.ca/int/jira/browse/WATER-999

I will add it as a comment.